stetho/src/main/java/com/facebook/stetho/inspector/elements/Document.java

```diff
@@ -140,6 +140,14 @@ public void setAttributesAsText(Object element, String text) {
     mDocumentProvider.setAttributesAsText(element, text);
   }
 
+  public void getElementStyle(int id, Accumulator<StyleInfo> styleInfoAccumulator) {
```

Comment:
I think we need to figure out what type of API we are really wanting to provide with `Document`. This method is essentially a utility method that can serve to confuse clients ("but I can do that myself via NodeDescriptor ... why would I use Document.getElementStyle?").

I also find it weird that this method is providing a access via an integer NodeID, but many other methods are using an Object element. Right now this seems to map to the direct needs of `CSS` which receives its request as an integer NodeID, but I think we can do better and provide a less confusing API overall.

Nothing directly actionable from this comment -- fodder for chin-scratching and later discussion.